net-misc/netstat-nat/files/netstat-nat-2.0-prefix.patch

```diff
@@ -0,0 +1,13 @@
+Replace default prefix
+
+--- a/GNUmakefile
++++ b/GNUmakefile
+@@ -5,7 +5,7 @@
+ DATE_EPOCH = LC_ALL=C date $(shell date -d @0 > /dev/null 2>&1 && echo "-d @" || echo "-r ")
+ SOURCE_DATE_EPOCH ?= $(shell git log -1 --no-show-signature --format=%at "netstat-nat.1.in")
+ MANUAL_DATE ?= $(shell $(DATE_EPOCH)$(SOURCE_DATE_EPOCH) +"%B %e, %Y")
+-PREFIX ?= /usr/local
```

Comment:
I don't think this is the right fix. We actually need to set `PREFIX=`${EPREFIX}`` in the ebuild instead in `emake` calls to work with Gentoo Prefix.